src/TraceEvent/Stacks/Linux/LinuxPerfScriptProcessNameBuilder.cs

```diff
@@ -16,36 +17,45 @@ internal sealed class LinuxPerfScriptProcessNameBuilder
                 ".NET Server GC"
             };
 
+        private readonly object _lock = new object();
```

Comment:
Maybe more descriptive name instead of `_lock` would be nicer?

Reply:
Sure.  How about `_dictionaryLock`?  I was going for a lock for a of the structures.

Reply:
`_dictionariesLock`, because it is guarding multiple dictionaries?

Reply:
Yes.  It's basically guarding all of the dictionaries, rather than having multiple locks.